GDJS/tests/tests/Extensions/TestRuntimeScene.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * A RuntimeScene that allows to test events side effects.
+ */
+gdjs.TestRuntimeScene = class TestRuntimeScene extends gdjs.RuntimeScene {
+
+  /**
+   * @param {gdjs.RuntimeGame} runtimeGame 
+   */
+  constructor(runtimeGame) {
+    super(runtimeGame);
+  }
+
+  /**
+   * @param {float} elapsedTime 
+   * @param {() => void} eventsFunction 
+   */
+  renderAndStepWithEventsFunction(elapsedTime, eventsFunction) {
+    this._eventsFunction = eventsFunction.bind(this, this);
```

Comment:
Prefer a anonymous function, slightly more readable that the old bind (and equally performant, as both create a new function)